docs/resources.html

```diff
@@ -214,25 +220,36 @@ <h2 id="improving-the-quality-of-reviews">Improving the quality of reviews</h2>
 				</li>
 				</ul>
 				<p>In addition to this, there are papers covering what <em>not</em> to do as a reviewer, for example humiliate the authors (<a href="https://doi.org/10.1080/17449642.2014.913341" target="_blank">Comer et al., 2014)</a> or be adversarial (<a href="https://doi.org/10.1145/1519103.1519122" target="_blank">Cormode, 2009</a>).</p>
-				<p>Unique issues arise for reviewing interdisciplinary research -- view and contribute to these conversations via this <a href="https://docs.google.com/document/d/1tTKbqEQ1gz5-8bZenyi4j2Tbnd0zvAuhmG805_S9NAs/edit" target="_blank">working document</a>.</p>
-				<p>For a more complete bibliography organized alphabetically, see the list <a href="https://docs.google.com/document/d/1ZsxtLZBV2gcZ4UrsMg4UKy8zdCS5YHWIgTwMuOJo-y8/edit?usp=sharing" target="_blank">here</a>.</p>
+				<p>Unique issues arise for reviewing interdisciplinary research -- view and contribute to these conversations via this <a href="https://docs.google.com/document/d/1tTKbqEQ1gz5-8bZenyi4j2Tbnd0zvAuhmG805_S9NAs/edit" target="_blank" title="Interdisciplinary review working document">working document</a>.</p>
+				<p>For a <a href="https://docs.google.com/document/d/1ZsxtLZBV2gcZ4UrsMg4UKy8zdCS5YHWIgTwMuOJo-y8/edit?usp=sharing" target="_blank">more complete bibliography organized alphabetically, see the list here</a>.</p>
+				<h2 id="incentivizing-reviews">Incentivising reviews</h2>
+				<p>The peer review system can be conceptualized as a “gift economy”, i.e. a system where previously published authors repay their debt for the labor that other scholars acting as editors and reviewers have invested in their manuscripts by acting as reviewers for others (<a href="LINK" target="_blank">TEXT</a>Kaltenbrunner et al., 2021). Despite this, it can be increasingly difficult to find reviewers for papers.</p>
```

Comment:
Something is missing here

Reply:
not sure what you mean -- see here, is there something missing that's in the Google doc that's not here? or something that should be in the Google doc and isn't?

https://docs.google.com/document/d/1FCVCbjoXReqclqYTUfBjdEJW_UCZ1ZHH/edit?rtpof=true

Reply:
It just says TEXT, which I assume is a placeholder